src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    Circle(Color color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: circle, area: " + getArea() + " sq. units, radius: "

```

Comment:
Avoid using direct `System.out.println` calls in your classes. Consider using a logger or passing a PrintStream object to the method for more flexible output handling.